### DISCUSSION TITLE
SWH from a certain altimetry pass?

### DISCUSSION THREAD

**Discussion by AndresSepulveda** [NOAA-EMC/WW3#845] [Q&A]
Hi,

We are running WW3 in an area with little to no buoy data and we would like to compare the results with a certain pass from Sentinel 3A and 3B. Is there a script that can be used to get the latest data?

Regards,

Andrés

**Comment by aliabdolali** [comment]
@AndresSepulveda This is not related to WW3 code itself.
I'd suggest to look at https://github.com/NOAA-emc/ww3-tools where there are scripts to download satellite altimeter's data
Alternatively, you can check noaa coastwatch https://coastwatch.noaa.gov/cwn/products/along-track-significant-wave-height-wind-speed-and-sea-level-anomaly-multiple-altimeters.html to grab 3A and 3B.
